src/PlusServer/vtkPlusOpenIGTLinkServer.cxx

```diff
@@ -1313,7 +1313,6 @@ igtl::MessageBase::Pointer vtkPlusOpenIGTLinkServer::CreateIgtlMessageFromComman
       replyStr << " Message=\"" << commandResponse->GetResultString() << "\"></CommandReply>";
       igtlMessage->SetMetaDataElement("Message", IANA_TYPE_US_ASCII, commandResponse->GetResultString());
 
-      igtlMessage->SetMetaDataElement("Result", IANA_TYPE_US_ASCII, (commandResponse->GetStatus() ? "true" : "false"));

```

Comment:
Was this change intentional?

Reply:
No, I don't know how that got deleted. I think I fixed it in the latest commit.